packages/concerto-core/lib/introspect/mapvaluetype.js

```diff
@@ -69,7 +69,8 @@ class MapValueType extends Decorated {
      */
     validate() {
         if (!ModelUtil.isPrimitiveType(this.type)) {
-            const decl = ModelUtil.getTypeDeclaration(this.ast.type.name, this.modelFile);
+
+            const decl = this.modelFile.getType(this.ast.type.name);
 
             // All declarations, with the exception of MapDeclarations, are valid Values.
```

Comment:
Have we got a test for enums and scalars as values?

Reply:
For Map Key [yes](https://github.com/accordproject/concerto/blob/9ff9b0c48a5d7944103e2882d3a5022d4cdbf464/packages/concerto-core/test/introspect/mapdeclaration.js#L167). Coverage for imported Scalars as Values I've just added..

Reply:
We don't support Enums as Map types (yet).